javascripts/background_scripts/embeded.js

```diff
@@ -0,0 +1,38 @@
+/*global chrome */
```

Comment:
If not, let's use the misspelling and do a global replace once everything is reviewed and merged.